### ISSUE TITLE
Patches not working

### ISSUE THREAD

**Issue by sushimehl** [mattcurrie/gb-save-states#49]
Hello and thanks for your work. I tried every rom I could get my hands on, but sadly "Legend of Zelda, The - Oracle of Seasons (E) (M5) [C][!].gbc.bsdiff" and also "Bomberman GB (U) [S][!].gb.bsdiff" don't seem to work. I've patched nearly every other game you got patches for (successully), but these two only give me a blank white screen... Thought I should report this.

**Comment by mattcurrie** [comment]
Hi there, thanks for your report.

I tried Bomberman GB (U) [S][!].gb and that one seems to be saving and loading fine. These are the hashes of the ROMs:
md5 before patching: 7e4c9c3620bea7b633394beb67e9680b
md5 after patching: 9dd8e8f09168c146aa28fef743a4e3ad

Also I tried Legend of Zelda, The - Oracle of Seasons (E) (M5) [C][!].gbc and that one was getting a blank white screen for me too. I've pushed a new patch, let me know if you run into any issues.
